README.md

```diff
@@ -34,7 +34,8 @@ Check out the website: https://linkfree.eddiehub.io
 Before contributing or adding a new feature, please make sure you have already installed the following tools:
 
 - [Git](https://git-scm.com/downloads)
-- [NodeJs](https://nodejs.org/en/download/)
+- [NodeJs](https://nodejs.org/en/download/) (Works with node - lts version v16.17.0)
+- Optional [NVM](https://github.com/nvm-sh/nvm): Switch node version - `nvm use` ( on windows `nvm use v16.17.0`), if this is not installed run `nvm install v16.17.0`.
```

Comment:
Just a few minor improvements. 🙂
```suggestion
- [NodeJs](https://nodejs.org/en/download/) (Works with Node LTS version v16.17.0)
- Optional [NVM](https://github.com/nvm-sh/nvm): Switch Node version by using `nvm use` (on Windows, use `nvm use v16.17.0`). If this is not installed, run `nvm install v16.17.0`.
```

Reply:
Thanks, Yes the English is better in this version